### ISSUE TITLE
Problems with non-ASCII strings from extra vars

### ISSUE THREAD

**Issue by mscherer** [ansible/ansible#10106]
See #10007 

```
ansible-playbook toto.yml -e "foo=ééé" 
```

with

```

---
- hosts: 127.0.0.1
    connection: local
    tasks:
    - debug: msg="plop {{ foo }}"
```

This result in the same problem, but with a different codepath.

**Comment by abadger** [comment]
Confirmed this is broken in v2 as well as v1.

**Comment by abadger** [comment]
Okay, v2 should be fixed.  I have an integration test for this as well.  Looking at v1 before committing that.

**Comment by abadger** [comment]
For v1, should be fixed by: 5b279c1c15fef88497d3806192da5d5024ebcc7e

# Closing This Ticket

Hi!

We believe recent commits (likely detailed above) should resolve this question or problem for you.

This will also be included in the next major release.

If you continue seeing any problems related to this issue, or if you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular
issue is resolved.

Thank you!
